backend/pages/models.py

```
@@ -60,6 +61,19 @@ def process_and_store_csv(self):
                 processed_data_frame, clean_sheet_name, sheet_title  = run_pipeline_for_sheet(xls, sheet_name, output_dir)
+                
+                if(clean_sheet_name in ADDITIONAL_PROCESSING_PIPELINE):
+                    processing_functions= ADDITIONAL_PROCESSING_PIPELINE[clean_sheet_name]
+                    df_processing = processed_data_frame
+                    
+                    for fn in processing_functions:
+                        df_processing = fn(df_processing)
+                    
+                    print("sheet slug:" + clean_sheet_name)
+                    print("additional processing required", df_processing)
+                    processed_data_frame = df_processing;
```

Ruff (0.8.2)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)